parent/pom.xml

```diff
@@ -638,7 +638,7 @@
                                     <version>[1.8,)</version>
                                 </requireJavaVersion>
                                 <DependencyConvergence />
-                                <requirePluginVersions />
+                                <!--<requirePluginVersions />-->
```

Comment:
why is this uncommented?

Reply:
sorry, it's from local build. Because of some strange reasons it fails locally.